components/gallery/GalleryItemDescription.vue

```diff
@@ -93,6 +91,36 @@
         >
       </div>
     </o-tab-item>
+
+    <!-- parent tab -->
+    <div v-if="parent">
+      <o-tab-item value="3" :label="$t('tabs.parent')" class="p-5">
+        <nuxt-link
+          :to="
+            urlOf({
+              id: parent?.nft.value?.id,
+              url: parentUrl,
+              chain: urlPrefix,
+            })
```

Comment:
I would pass it as as simple string literal or computed value,

We can think of some urlComposables